openqa-advanced-retrigger-jobs

```diff
@@ -8,9 +8,47 @@ result="${result:-"result='incomplete'"}"
 additional_filters="${additional_filters+" and $additional_filters"}"
 comment="${comment:-""}"
 dry_run="${dry_run:-"0"}"
-[ "$dry_run" = "1" ] && client_prefix="echo"
-# shellcheck disable=SC2029
-for i in $(ssh "$host" "sudo -u geekotest psql --no-align --tuples-only --command=\"select id from jobs where (${worker_string}${result} and clone_id is null and t_finished >= '$failed_since'$additional_filters);\" openqa"); do
-    $client_prefix openqa-cli api --host "$host" -X POST jobs/"$i"/restart
-    [ -n "$comment" ] && $client_prefix openqa-cli api --host "$host" -X POST jobs/"$i"/comments text="$comment"
-done
+
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
```

Comment:
Please provide an example call and/or a list of arguments (that are supplied via env in this case I guess)